src/prod_components/PhotoEditor.js

```diff
@@ -14,7 +14,16 @@ const PhotoEditor = forwardRef((props, ref) => {
   const [errorVisible, setErrorVisible] = useState(false)
 
   const [imgSrc, setImgSrc] = useState(null)
-  const [crop, setCrop] = useState({ aspect: 1 / 1, maxWidth: 1000, minWidth: 60})
+  const [crop, setCrop] = useState({
+    aspect: 1 / 1,
+    unit: '%',
+    width: 80,
+    height: 80,
+    x: 10,
+    y: 10,
+    maxWidth: 1000,
+    minWidth: 60
+  })
```

Comment:
I suppose this is the initial crop area?

Please reply to this comment or explain in the code what it is. Then I can approve this change 👍